client/capstone-app/src/App.js

```diff
@@ -1,14 +1,24 @@
 import './App.css';
-import Forms from"./component/form"
-import Navbar from"./component/Navbar"
+import Forms from"./component/form";
+import Navbar from"./component/Navbar";
+import Datas from './component/body';
+import DocList from './component/DocList';
+import PaymentPage from './component/Pay';
+import DocForm from './component/DocForm';
+import DocInfo from "./component/DocInfo";
 import { Route,Routes
  } from 'react-router-dom';
 function App() {
   return (
     <div className="App">
       <Navbar/>
       <Routes>
+        <Route path='/' element={<Datas/>}></Route>
         <Route path='/Forms' element={<Forms/>}></Route>
```

Comment:
As per general naming convection, the path's should be in `snake_case`. Please change them to `snake_case`. To know more about naming convection, please read this -> https://climbtheladder.com/10-react-naming-conventions-best-practices/